ibis/backends/bigquery/tests/unit/snapshots/test_compiler/test_to_timestamp_timezone/out.sql

```diff
@@ -1,3 +1,3 @@
 SELECT
-  parse_timestamp('%F %Z', CONCAT(`t0`.`date_string_col`, ' America/New_York'), 'UTC') AS `StringToTimestamp_StringConcat_'%F %Z'`
+  parse_timestamp('%F %Z', CONCAT(`t0`.`date_string_col`, ' America/New_York'), 'UTC') AS `StringToTimestamp_StringConcat_date_string_col_' America_New_York'_'%F %Z'`
```

Comment:
I think this is just ugly, not an issue, since bigquery quotes with backticks, yeah?

Reply:
Yeah, it's gross, but it should be fine. I'll add a `system` test to verify that the column name is allowed.

Reply:
Done, and passing:

```
cloud in 🌐 falcon in …/ibis on  fix-duplicate-projections is 📦 v9.1.0 via 🐍 v3.10.14 via ❄️   impure (ibis-3.10.14-env)
❯ pytest ibis/backends/bigquery/tests/system/test_client.py::test_complex_column_name -q
.                                                                                                                                                                                      [100%]
1 passed in 4.88s
```